rules/cloud/aws/cloudtrail/aws_lambda_function_url.yml

```
@@ -0,0 +1,25 @@
+title: AWS Created Malicious Lambda Function URL Configuration
+id: ec541962-c05a-4420-b9ea-84de072d18f4
+status: experimental
+description: |
+  Detects when a user creates a Lambda function URL configuration, which could be used to expose the function to the internet and potentially allow unauthorized access to the function's IAM role for AWS API calls.
+  This could give an adversary access to the privileges associated with the Lambda service role that is attached to that function.
+references:
+    - https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunctionUrlConfig.html
```

```suggestion
    - https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunctionUrlConfig.html
    - https://cloud.hacktricks.xyz/pentesting-cloud/aws-security/aws-privilege-escalation/aws-lambda-privesc
    - https://www.wiz.io/blog/how-to-set-secure-defaults-on-aws
```
Add some ref for better understanding